brave/brave-core
# Storybook: Created apps-list-component

Resolves <https://github.com/brave/brave-browser/issues/14955>

Created an apps list component and added it too storybook.
Also created a reusable search bar component that was needed for the apps panel.

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. `npm run storybook`
2. Confirm that the storybook shows new component.
3. Check functionality of the new component.

https://user-images.githubusercontent.com/40611140/113009437-2d0c3100-9135-11eb-8fbc-46d70c545318.mov
